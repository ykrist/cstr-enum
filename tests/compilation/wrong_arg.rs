use cstr_enum::*;

#[derive(AsCStr)]
enum Enum0 {
  #[cstr(name=0)]
  A,
  B,
  C
}

#[derive(AsCStr)]
enum Enum1 {
  #[cstr(foo="egg")]
  A,
  B,
  C
}

fn main() {

}