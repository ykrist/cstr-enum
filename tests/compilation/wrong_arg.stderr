error: expected string literal
 --> $DIR/wrong_arg.rs:5:15
  |
5 |   #[cstr(name=0)]
  |               ^

error: invalid named argument
  --> $DIR/wrong_arg.rs:13:10
   |
13 |   #[cstr(foo="egg")]
   |          ^^^
